dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: jitaccess-ops-phac-alpha-canada-ca-ns
+  namespace: alpha-dns
+  labels:
+    ownerName: "Eric Malenfant"
+    ownerContact: "eric.richard-malenfant@phac.aspc.gc.ca"
+    gitHubRepo: "JIT-IaC"
+spec:
+  name: "jitaccess.ops.phac.alpha.canada.ca."
+  type: "NS"
+  ttl: 300
+  managedZoneRef:
+    external: phac-alpha-canada-ca

```

I think this needs to be 
```suggestion
external: ops-phac-alpha-canada-ca-zone 
```

---

Done